include/xtensor-blas/xlinalg.hpp

```diff
@@ -1464,9 +1461,6 @@ namespace linalg
 
         blas_index_t rank;
 
-        int info = lapack::gelsd(dA, db, s, rank, rcond);
```

Comment:
it was used here?! :)

Reply:
Sorry, I was a bit too brutal. Should have taken coffee first ;)